Plots/Contours/NCL_eof_1_1.py

```diff
@@ -0,0 +1,320 @@
+    gvutil.add_lat_lon_ticklabels(ax)
+
+    return cplot, ax
+
+
+###############################################################################
+# Plot (1): Draw a contour plot for each EOF
+
+# Generate figure and axes using Cartopy projection  and set figure size (width, height) in inches
+fig, axs = plt.subplots(neof, 1, constrained_layout=True, subplot_kw={"projection": ccrs.PlateCarree()}, figsize=(6, 9))
+
+# Add multiple axes to the figure as contour and contourf plots
+for i in range(neof):
+
+    eof_single = eof.sel(evn=i)
+
+    # Create contour plot for the current axes
+    cplot, axs[i] = make_contour_plot(axs[i], eof_single)
+
+    # Use geocat.viz.util convenience function to add titles to left and right of the plot axis.
+    pct = eof.pcvar[i]
+    gvutil.set_titles_and_labels(axs[i], lefttitle=f'EOF {i + 1}', lefttitlefontsize=10,
+                                 righttitle=f'{pct:.1f}%', righttitlefontsize=10)
+
+# Add horizontal colorbar
+cbar = plt.colorbar(cplot, ax=axs, orientation='horizontal', shrink=0.6)
+cbar.ax.tick_params(labelsize=8)
+
+# Set a common title
+axs[0].set_title(f'SLP: DJF: {yearStart}-{yearEnd}', fontsize=14, y=1.12)
+
+# plt.savefig('test.png')
+plt.savefig('eof_1_1_lg.png')
+
+# Show the plot
+plt.show()
+
+
+###############################################################################
+# Utility function:
+
+# Define a utility function for creating a bar plot.
+
+def make_bar_plot(ax, dataset):
+
+    years = list(dataset.time.dt.year)
+    values = list(dataset.values)
+    colors = ['blue' if val < 0 else 'red' for val in values]
+
+    ax.bar(years, values, color=colors, width=1.0, edgecolor='k', linewidth=0.5)
+    ax.set_ylabel('Pa')
+
+    # Use geocat.viz.util convenience function to add minor and major tick lines
+    gvutil.add_major_minor_ticks(ax, x_minor_per_major=4, y_minor_per_major=5, labelsize=8)
+
+    # Use geocat.viz.util convenience function to set axes tick values
+    gvutil.set_axes_limits_and_ticks(ax, xticks=np.linspace(1980, 2000, 6), xlim=[1978.5, 2003.5])
+
+    return ax
+
+
+###############################################################################
+# Plot (2): Produce a bar plot for each EOF.
+
+# Generate figure and axes using Cartopy projection and set figure size (width, height) in inches
+fig, axs = plt.subplots(neof, 1, constrained_layout=True, figsize=(6, 7.5))
+
+# Add multiple axes to the figure as bar-plots
+for i in range(neof):
+
+    eof_single = eof_ts.sel(neval=i)
+
+    axs[i] = make_bar_plot(axs[i], eof_single)
+    pct = eof.pcvar[i]
+    gvutil.set_titles_and_labels(axs[i], lefttitle=f'EOF {i + 1}', lefttitlefontsize=10,
+                                 righttitle=f'{pct:.1f}%', righttitlefontsize=10)
+
+# Set a common title
+axs[0].set_title(f'SLP: DJF: {yearStart}-{yearEnd}', fontsize=14, y=1.12)
+
+# plt.savefig('test.png')
```

Comment:
ibid

Reply:
Yes, thanks a lot for catching this! Removed it with a new commit.